HTTPServerProject.Tests/HTTPServerWriteTests/TestWriteStreams.cs

```diff
@@ -0,0 +1,64 @@
+using HTTPServerWrite.Streams;
+
+namespace HTTPServerWriteTests.Streams
+{
+    public class UnitTestsForWriteStreams
+    {
+        [Fact]
+        public void WriterIsClosedTest()
```

Comment:
What about when you don't call `Close` - it shouldn't have called `close` - correct?

Reply:
Oh but this is another test for a dummy implementation. You generally don't need those.

Reply:
Sounds good.